src/features/dataset-controls/DatasetControls.js

```diff
@@ -34,7 +34,7 @@ const authTypes = [
 
 var host = window.location.host;
 var hostname = window.location.hostname;
-var port = '80';
```

Comment:
this is an ephemeral port - it shouldn't be hardcoded to this

Reply:
note this how it gets the port for poseidon: https://github.com/CyberReboot/CRviz/blob/8eb5096cceeea70b18ea919664e619b1379566d0/src/features/dataset-controls/DatasetControls.js#L40